Geometry/GEMGeometry/src/ME0Chamber.cc

```diff
@@ -78,3 +78,23 @@ const ME0EtaPartition* ME0Chamber::etaPartition(int isl) const {
   }
   return 0;
 }
+
+float ME0Chamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction ) const {
```

Comment:
The "momentum phi" measurement depends on the eta of the object. The reason being that it is the momentum of the particle in the reference frame of the particle in the chamber. 

This can be seen in the attached image. The phi direction in the frame of the particle depends on the eta measurement. All we can say precisely is that the segment or track transversed n strips, which means theres a set phi between the inner and outer layer. To say that the momentum has a certain phi value requires precision in the eta coordinate.
![evernote snapshot 20170403 130116](https://cloud.githubusercontent.com/assets/3338207/24629401/bb20fa88-186d-11e7-9649-6d2628821490.png)